src/tests/functional/plugin/cpu/single_layer_tests/transpose.cpp

```diff
@@ -1,323 +1,327 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+// Since the Transpose ngraph operation is converted to the transpose node, we will use it in the transpose test
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+
+typedef std::tuple<
+        InputShape,                    // Input shapes
+        std::vector<size_t>,                // Input order
+        InferenceEngine::Precision,         // Net precision
+        std::string,                        // Target device name
+        std::map<std::string, std::string>, // Additional network configuration
+        CPUSpecificParams> TransposeLayerCPUTestParamSet;
+
+class TransposeLayerCPUTest : public testing::WithParamInterface<TransposeLayerCPUTestParamSet>,
+                              public ov::test::SubgraphBaseTest, public CPUTestsBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<TransposeLayerCPUTestParamSet> obj) {
+        Precision netPrecision;
+        InputShape inputShapes;
+        std::vector<size_t> inputOrder;
+        std::string targetDevice;
+        CPUSpecificParams cpuParams;
+        std::map<std::string, std::string> additionalConfig;
+        std::tie(inputShapes, inputOrder, netPrecision, targetDevice, additionalConfig, cpuParams) = obj.param;
+
+        std::ostringstream result;
+        result << "IS=" << CommonTestUtils::partialShape2str({inputShapes.first}) << "_";
+        result << "TS=" << CommonTestUtils::vec2str(inputShapes.second) << "_";
+        result << "inputOrder=" << CommonTestUtils::vec2str(inputOrder) << "_";
+        result << "netPRC=" << netPrecision.name() << "_";
+        result << "trgDev=" << targetDevice;
+        result << CPUTestsBase::getTestCaseName(cpuParams);
+        return result.str();
+    }
+protected:
+    void SetUp() override {
+        Precision netPrecision;
+        InputShape inputShapes;
+        std::vector<size_t> inputOrder;
+        CPUSpecificParams cpuParams;
+        std::map<std::string, std::string> additionalConfig;
+        std::tie(inputShapes, inputOrder, netPrecision, targetDevice, additionalConfig, cpuParams) = this->GetParam();
+        configuration.insert(additionalConfig.begin(), additionalConfig.end());
+
+        inType = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+        outType = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+
+        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+
+        selectedType = makeSelectedTypeStr("unknown", inType);
+
+        targetStaticShapes.reserve(inputShapes.second.size());
+        for (const auto& staticShape : inputShapes.second) {
+            targetStaticShapes.push_back({staticShape});
+        }
+        inputDynamicShapes = { inputShapes.first };
```

Comment:
It is better to use `init_input_shapes({inputShape});`

Reply:
changed, Thanks!